requirements.txt

```diff
@@ -16,7 +16,7 @@ netifaces
 numba
 numpy
 pyephem
-pyjwt
+pyjwt==2.0.1
```

Comment:
Ideally we should instead update the pin in katsdpdockerbase.

Reply:
OK, should I do that first or as a separate step?

Reply:
I'd say prepare the PR for it, and merge both at the same time.